tardis/workflows/standard_simulation.py

```diff
@@ -0,0 +1,247 @@
+import logging
+
+from tardis.io.util import HDFWriterMixin
+from tardis.simulation.base import PlasmaStateStorerMixin
+from tardis.spectrum.luminosity import (
+    calculate_filtered_luminosity,
+)
+from tardis.util.base import is_notebook
+from tardis.visualization import ConvergencePlots
+from tardis.workflows.simple_simulation import SimpleSimulation
+
+# logging support
+logger = logging.getLogger(__name__)
+
+
+class StandardSimulation(
+    SimpleSimulation, PlasmaStateStorerMixin, HDFWriterMixin
+):
+    hdf_properties = [
+        "simulation_state",
+        "plasma_solver",
+        "transport_solver",
+        "iterations_w",
+        "iterations_t_rad",
+        "iterations_electron_densities",
+        "iterations_t_inner",
+        "spectrum_solver",
+    ]
+
+    def __init__(
+        self,
+        configuration,
+        enable_virtual_packet_logging=False,
+        log_level=None,
+        specific_log_level=None,
+        show_progress_bars=False,
+        show_convergence_plots=False,
+        convergence_plots_kwargs={},
+    ):
+        self.show_progress_bars = show_progress_bars
+        self.log_level = log_level
+        self.specific_log_level = specific_log_level
+        self.enable_virtual_packet_logging = enable_virtual_packet_logging
+
+        SimpleSimulation.__init__(self, configuration)
+
+        # set up plasma storage
+        PlasmaStateStorerMixin.__init__(
+            self,
+            iterations=self.total_iterations,
+            no_of_shells=self.simulation_state.no_of_shells,
+        )
+
+        # Convergence plots
+        if show_convergence_plots:
+            if not is_notebook():
```

Comment:
A think some of these plotting branches could be separate methods to make the init a bit more readable